src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -284,6 +285,95 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
     }
     
+    private void testEditVisibilityAction() throws Exception {
+
+        ______TS("edit question 1 to Giver's team members and Giver");
```

Comment:
@Sumei1009, the test is quite excessive. You should only be testing that the behaviour of <code>Hide "Recipient's team members" for recipient type = "Giver's team member and giver"</code> is followed. 

Also, instead of editing the existing question, will it be easier to just add a new question and edit its details? That should be quicker too.